microsoft-edge/devtools-guide-chromium/network/reference.md

```diff
@@ -350,7 +350,7 @@ Right-click the header of the Requests table and select an option to hide or sho
 
 #### Add custom columns
 
-To add a custom column to the Requests table, right-click the header of the Requests table and then select **Response Headers** > **Manage Header Columns**.
+To add a custom column to the Requests table, right-click the header of the Requests table and then select **Response Headers** > **Manage Header Columns**. In the **Manage Header Columns** popup window, click **Add Custom Header**. Enter the custom header name and click **Add** to add a new column to the requests table. 
```

Comment:
```suggestion
To add a custom column to the Requests table, right-click the header of the Requests table and then select **Response Headers** > **Manage Header Columns**. The **Manage Header Columns** popup window opens.  Click the **Add custom header** button, enter the custom header name, and then click **Add**. 
```
brought wording closer to standard action > result UI format.  Lowercased button name per UI label.  

gh rendered section (commit-specific):
https://github.com/MicrosoftDocs/edge-developer/blob/447e4f1c612cee93f1fc5f3cab2a8785cf86098c/microsoft-edge/devtools-guide-chromium/network/reference.md#add-custom-columns